web-ui/src/main/resources/catalog/locales/fr-search.json

```diff
@@ -186,7 +186,7 @@
     "orTypeAServiceUrl": "saisir l'URL d'un service {{type}}",
     "addAllLayersToMap": "Ajouter toutes les {{number}} couches à la carte",
     "downloadsAndResources": "Téléchargements et liens",
-    "sourceCatalog": "Fourni par",
+    "sourceCatalog": "Catalogue d'origine",
```

Comment:
JSON translation files must be made in https://www.transifex.com/geonetwork/signin/?next=/geonetwork/core-geonetwork/translate/#fr. See https://geonetwork-opensource.org/manuals/4.0.x/en/contributing/translating.html